src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -54,19 +54,21 @@ public JsonAdaptedPerson(Person source) {
         phone = source.getPhone().value;
         email = source.getEmail().value;
         address = source.getAddress().value;
-        tagged.addAll(source.getTags().stream()
+        tags.addAll(source.getTags().stream()
                 .map(JsonAdaptedTag::new)
                 .collect(Collectors.toList()));
     }
 
     /**
-     * Converts this Jackson-friendly adapted person object into the model's {@code Person} object.
+     * Converts this Jackson-friendly adapted person object into the model's
+     * {@code Person} object.
```

Comment:
Unrelated change?

Reply:
I have updated my formatter to only format source changes.